app/workers/activitypub/delivery_worker.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class ActivityPub::DeliveryWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: 'push', retry: 5, dead: false
+
+  HEADERS = { 'Content-Type' => 'application/activity+json' }.freeze
```

Comment:
ActivityPub does not specify the content type for Server-to-Server interactions, but https://w3c.github.io/activitypub/#client-to-server-interactions implies that the *primary* content type of ActivityPub is `application/ld+json; profile="https://www.w3.org/ns/activitystreams"`, and that all clients need to use that content type and that's the only one that's a `must` in the spec (`activity+json` is a `should`)